### ISSUE TITLE
Flow Blockchain function can be called succesfully from the CLI

### ISSUE THREAD

**Issue by lukaswolfrum** [amosproj/amos2022ss07-nft-playbook#44]
## User story
1. As a power user 
2. I want / need to select the flow blockchain in the blockchain menu
3. So that i can easily mint on flow blockchain

## Acceptance criteria
* flow is selectable in the blockchain-selection menu
* the selection calls the flow blockchain function from backend correctly
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
